perf/istio-install/gke_cluster_resources/cluster.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: identity.cnrm.cloud.google.com/v1alpha2
+kind: IdentityNamespace
+metadata:
+  name: default
+  namespace: "test-istio-237517" # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.core.project","value":"test-istio-237517"}}}
```

Comment:
This looks like an actual project -- I'd set it to something like `[PROJECT_ID]` before committing.

Reply:
updated